index.html

```diff
@@ -29,8 +29,8 @@ <h1 class="title">
                 <h2 class="subtitle">
                     web apps shared in a chat
                 </h2>
-                <p>
-                    no logins, no coins, no platforms 
+                <p class="blurb">
+                    build interactive and collaborative experiences without sacrificing privacy using html5, then share with friends by simply sending a message.
```

Comment:
There is a lot of messaging from "privacy oriented" projects from the web3/coin folks, e.g. https://getsession.org/ -- the whole page does not mention that things are based on coins but promises the world.  The three "no's" are meant to interest those who are skeptical of central platforms *and* coins (and those who recgonize that all things associated with "logins" are obnoxious).  It's true that the three no's don't say what it is, but with the 5-7 people i introduced to the page the last week, they all smirked and liked the tag line.  So i'd preserve more descriptive things for the cookbook or later sections.

Reply:
@hpk42 how does it look now with both?